cips/cip-14.md

```diff
@@ -213,7 +213,18 @@ This proposal is backwards-incompatible because it is state-machine breaking. Th
 
 ## Test Cases
 
-TBA
+The following test cases are available in the [ibc-go e2e repository.](https://github.com/cosmos/ibc-go/tree/main/e2e/tests/interchain_accounts)
+
+- Registration of an interchain account
+- Transfer funds from interchain account to a different account on the same chain using an unordered or ordered channel
+- A failed transfer of funds from interchain account to a different account on the same chain due to insufficient funds
+- Transfer funds from interchain account to a different account on the same chain after an ordered channel closure
+- A transfer of funds from an interchain account to a different account on the same chain using a gov controller
+- A transfer of funds from an interchain account to a different account on the same chain using a groups controller
+- A transfer of funds from an interchain account to a different account on the same chain using an incentivised IBC packet
+- Registration of an interchain account, and transfer of funds from an interchain account to a different account on the same chain through the localhost client, and after reopening a channel
```

Comment:
This test case sounds like it's testing multiple scenarios. Would it make sense to split into multiple test cases? Also what's a localhost client?

Reply:
can split, localhost / loopback is a client type that enables you to interact with a chain locally through IBC. This test case would only be relevant for if you have the client type enabled, and requires ibc-go v7+ so perhaps I can just remove reference to these test cases?

Reply:
I found the docs here: https://github.com/cosmos/ibc-go/tree/v6.2.1/modules/light-clients/09-localhost

> This test case would only be relevant for if you have the client type enabled

Based on [this search](https://github.com/search?q=repo%3Acelestiaorg%2Fcelestia-app+09-localhost&type=code) it looks like celestia-app doesn't have the `09-localhost` client type imported or enabled.

> and requires ibc-go v7+

Celestia-app currently uses ibc-go v6 (see [this line](https://github.com/celestiaorg/celestia-app/blob/266010890efdedae7496da37ec2e8a63d38dfce4/go.mod#L17)) so seems like we can remove this test case.